.github/lock.yml

```diff
@@ -0,0 +1,29 @@
+# Number of days of inactivity before a closed issue or pull request is locked
```

Comment:
Doesn't this mean that if people comment/spam on closed issues then the issue remains unlocked? I thought we wanted `Number of days since closure`.

Reply:
Probably. But it's better than nothing.

Reply:
The current implementation does use number of days since last updated:

https://github.com/dessant/lock-threads/blob/8b1ae9c84a4219662ad2dd4efec00b0888b23cf6/src/lock.js#L69

Using last closed for the query instead should be possible with modifications to the bot:

https://help.github.com/en/articles/searching-issues-and-pull-requests#search-by-when-an-issue-or-pull-request-was-closed

To keep existing auto-close behavior we should see if we can get an upstream change to the bot to support filtering based on closed date.

Reply:
I think it should be possible.

Reply:
@mattclay @mkrizek https://github.com/dessant/lock-threads/issues/14#issuecomment-508758023